python/serverless-backend/README.md

```diff
@@ -0,0 +1,78 @@
+# Serverless-backend for your UI / Frontend example
+
+This is a project to deploy a serverless backend for your current project on AWS with the AWS Cloud Development Kit.
+
+
+
+This project also demonstrates how to build a serverless backend application that will allow users to make authenticated API calls with API Gateway and Cognito and have the request be processed by a Lambda. Will use S3 to store images/files and DynamoDB to store any metadata needed. 
```

Comment:
Clarify and shorten this: Tell me what I'll be able to do when I've deployed this stack & what is happening behind the scenes.